scripted_render_pipeline/importer/__main__.py

```diff
@@ -12,36 +13,40 @@
 
 # render properties
 HOST = "https://sonic.tnw.tudelft.nl"
-OWNER = "rlane"
-# PROJECT = "20191101_ratpancreas_partial_partial_test"
-PROJECT = "20230523_singleholder_Earthworm_03_partial_partial_test"
+OWNER = "akievits"
+PROJECT = "20231107_MCF7_UAC_test"
+CORRECTIONS_DIR = "postcorrection"
 
 # script properties
 PARALLEL = 40  # read this many images in parallel to optimise io usage
 CLOBBER = True  # set to false to fail if data would be overwritten
 Z_RESOLUTION = 100  # the thickness of sections
-REMOTE = True  # set to false if ran locally
+REMOTE = False  # set to false if ran locally
 NAS_SHARE_PATH = pathlib.Path.home() / "shares/long_term_storage"
 SERVER_STORAGE_PATH_STR = "/long_term_storage/"
 PROJECT_PATH = (
     (NAS_SHARE_PATH if REMOTE else pathlib.Path(SERVER_STORAGE_PATH_STR))
     # / "thopp/20191101_rat-pancreas_partial"
     #  / "rlane/SECOM/projects/20191101_rat-pancreas_partial"
-    / "thopp/20230523_singleholder_Earthworm_03/ROA-1"
+    / f"akievits/FAST-EM/tests/{PROJECT}"
 )
 MIPMAP_TYPE = "FASTEM"  # "CLEM"
 # for fastem datasets only
 USE_POSITIONS = True  # use the automated stitching results
+MULTIPLE_SECTIONS = True # Set to False for a single section
```

Comment:
setting this to false would raise a NameError at line 49

Reply:
Allright, then set PROJECT_PATHS either to the natsorted list or to "None"?